src/mastercard-auth.js

```diff
@@ -25,7 +25,7 @@ module.exports = function (context) {
     
     // The plugin is only active for certain URIs. By default, it intercepts API calls 
     // sent to "mastercard.com" URIs.
-    const appliesTo = mastercard.appliesTo ? mastercard.appliesTo : [ "mastercard.com" ]
+    const appliesTo = mastercard.appliesTo ? mastercard.appliesTo : [ "mastercard.com", "ethocaweb.com" ]
```

Comment:
Because Mastercard acquired Ethoca, it makes sense to have "api.ethocaweb.com" in the list.

The problem I see is the line of code you changed is only here for backward compatibly reasons, when 
 someone doesn't have the new `appliesTo` section in their environment.

I think a better approach for the plugin to send requests sent to "api.ethocaweb.com" would be to add "api.ethocaweb.com" to the default workspace here: https://github.com/Mastercard/insomnia-plugin-mastercard-auth/tree/master/workspace

And maybe in the README.md here: https://github.com/Mastercard/insomnia-plugin-mastercard-auth#configuration

Reply:
@jaaufauvre when the person doesnt have the appliesTo property, it will default to see if the HOST contains either mastercard.com / api.ethocaweb.com right?

updated both the pre-constructed workspace and the readme

Reply:
@yongjian85, when `appliesTo` doesn't exist (users with an old config) we simply define it at runtime (`appliesTo = [ "mastercard.com" ]`). What follows should be the same.